APIMatic.Core/ApiCall.cs

```diff
@@ -82,6 +82,14 @@ public ApiCall<Request, Response, Context, ApiException, ReturnType, ResponseTyp
             return this;
         }
 
+        public ApiCall<Request, Response, Context, ApiException, ReturnType, ResponseType> RequestBuilder(RequestBuilder _requestBuilder)
+        {
+            requestBuilder = _requestBuilder;
+            return this;
+        }
+
+        public RequestBuilder GetRequestBuilder() => requestBuilder;
```

Comment:
Can we convert this into a property? what's the reason for making it public? can we keep it internal?